File: step_by_step/step_1/2588.py
# 곱셈

A = input()
B = input()
int_A = int(A)
int_B = int(B)

line3 = int_A * (int_B % 10)
line4 = int_A * ((int_B // 10) % 10)
line5 = int_A * (int_B // 100)
line6 = line3 + (line4*10) + (line5*100)

print(line3)
print(line4)
print(line5)
print(line6)

File: step_by_step/step_1/c/2588.c
#include <stdio.h>

int main(void){
    int A, B;
    scanf("%d %d", &A, &B);

    int line3 = A * (B % 10);
    int line4 = A * ((B / 10) % 10);
    int line5 = A * (B / 100);
    int line6 = line3 + (line4*10) + (line5*100);

    printf("%d\n", line3);
    printf("%d\n", line4);
    printf("%d\n", line5);
    printf("%d\n", line6);
    return 0;
}